File: 백준/Bronze/28295. 체육은 코딩과목 입니다/체육은 코딩과목 입니다.cs
using System;

public class Program
{
    public static void Main(string[] args)
    {
        int total = 0;

        for (int i = 0; i < 10; i++)
        {
            int a = int.Parse(Console.ReadLine());
            total += a;
        }

        if (total % 4 == 0)
        {
            Console.WriteLine("N");
        }
        else if (total % 4 == 1)
        {
            Console.WriteLine("E");
        }
        else if (total % 4 == 2)
        {
            Console.WriteLine("S");
        }
        else if (total % 4 == 3)
        {
            Console.WriteLine("W");
        }
    }
}

File: 백준/Bronze/28295. 체육은 코딩과목 입니다/체육은 코딩과목 입니다.py
total = 0

for _ in range(10):
    a = int(input())
    total += a

if total % 4 == 0:
    print("N")
elif total % 4 == 1:
    print("E")
elif total % 4 == 2:
    print("S")
elif total % 4 == 3:
    print("W")
